website/www/site/layouts/shortcodes/playground.html

```diff
@@ -10,10 +10,13 @@
 limitations under the License. See accompanying LICENSE file.
 */}}
 
-<iframe
-    src="https://frontend-beta-dot-apache-beam-testing.appspot.com/embedded?enabled={{ .Get 0 }}&example={{ .Get 1 }}&code={{ .Get 2 }}"
-    width="100%"
-    height="{{ .Get 3 }}"
-    class="code-snippet playground"
-    allow="clipboard-write">
-</iframe>
+<div id="playgroundIframeContainer">
+    <iframe
+        src="https://frontend-beta-dot-apache-beam-testing.appspot.com/embedded?enabled={{ .Get 0 }}&example={{ .Get 1 }}&code={{ .Get 2 }}"
+        width="100%"
+        height="{{ .Get 3 }}"
+        id="{{ .Get 4 }}"
```

Comment:
What is the use for this ID? Is it used outside of the shortcode? If not, can it be generated here or omitted?

Reply:
removed id

Reply:
Playground now has subdomain https://play.beam.apache.org/ - would be nice to update

Reply:
done